.github/workflows/update-snapshots.yml

```diff
@@ -28,7 +32,6 @@ jobs:
         with:
           repository: ${{ github.event.pull_request.head.repo.full_name }}
           ref: ${{ github.event.pull_request.head.ref }}
-          token: ${{ secrets.GIX_CREATE_PR_PAT }}
```

Comment:
Better to keep minimal permissions as I don't think the PAT is required here. It seems like this action is just pushing a doctor image, not creating a PR, so the above permissions should be enough.